tests/tests_e3/net/smtp/main_test.py

```diff
@@ -21,3 +21,46 @@ def test_sendmail():
             from_addr,
             to_addresses,
             msg_as_string)
+
+def test_sendmail_onerror():
+    from_addr = 'e3@example.net'
+    to_addresses = ['info@example.net', 'info@example.com']
+    msg_as_string = 'test mail content'
+    msg_size_excceed = "A"*1200
+    result = e3.net.smtp.sendmail(
+        from_addr,
+        to_addresses,
+        msg_size_excceed,
+        ['smtp.localhost'],
+        max_size=8/1024)
+    assert result == False
+
+    with mock.patch('smtplib.SMTP') as mock_smtp:
+        mock_smtp.side_effect = smtplib.SMTPException()
+        result = e3.net.smtp.sendmail(
+            from_addr,
+            to_addresses,
+            msg_as_string,
+            ['smtp.localhost'])
+        assert result == False
+
```

Comment:
You should never have two empty lines in an indented block. But apparently we’re not running any style checks on our tests. I think I’ll add it.